ts/src/profile-serializer.ts

```diff
@@ -156,9 +156,7 @@ function serialize<T extends ProfileNode>(
         profLoc = sourceMapper.mappingInfo(profLoc);
       }
     }
-    const keyStr = `${node.scriptId}:${profLoc.line}:${profLoc.column}:${
-      profLoc.name
-    }`;
+    const keyStr = `${node.scriptId}:${profLoc.line}:${profLoc.column}:${profLoc.name}`;
```

Comment:
This seems an unrelated change, is this expected?

Reply:
The version of gts used seem to change with this update; I had to run `npm run fix` so unit tests passed.

Noted this in the description.